Cardano Threat Intelligence/Weaknesses/CTI-2023-ADA-51-01.md

```diff
@@ -0,0 +1,41 @@
+# CTI-2023-ADA-51-01: Infinite Minting
+
+## Author
+
+Alasdair Hill, Certification Engineer, IO Global
+
+## Short Summary
+
+Infinite minting is an exploit where an attacker manages to mint a numerous amount of tokens in a way that is unintended by the DApp. This exploit has severe consequences and has caused tokens to completely lose all of their value.
+
+## Description
+
+Many DApps have validators that govern the minting of tokens. An infinite minting attack occurs when an attacker can mint numerous tokens in a way that is not expected by the validator. This allows an attacker to mint a potentially infinite amount of tokens that weren't meant to be minted. Typically this type of attack is used to generate a large number of tokens which are then dumped. This potentially generates millions of dollars for the attacker whilst completely devaluing the generated token. 
+
+An infinite minting attack is severe as it can affect not only the DApp where the token is minted but also all other DApps that accept the minted token. To date this type of attack has been used to steal over a billion dollars with some top examples detailed [here](https://pontem.network/posts/top-crypto-infinite-mint-hacks). 
+
+## Risk Assessment
+
+- **Likelihood:** high
+
+- **Difficulty:** high
+
+- **Impact:** high
+
+- **Severity:** high
```

Comment:
I think it's not possible to be severity "High" with a difficulty of "High"

Reply:
I think high is the correct severity as it is the middle one